scripts/talos/README.md

```diff
@@ -0,0 +1,57 @@
+# Talos
+
+## Description
+
+Talos is a bumper bot for Psammead packages.
+
+## Usage
```

Comment:
Can we make it clearer that Talos runs automatically and that you do not have to always run it manually initially?